### PR TITLE
:sparkles: add mqtt topics to options

### PR DESCRIPTION
**Pull request by skeeey**
## Related issue(s)

Fixes #

### CODE REVIEW

File: pkg/cloudevents/generic/types/types.go [hunk 1]
```diff
@@ -82,6 +82,18 @@ const (
 	Deleted ResourceAction = "DELETED"
 )
 
+// Topics represents required messaging system topics for a source or agent
+type Topics struct {
+	// SpecTopic is a topic for resource spec
+	SpecTopic string `json:"specTopic" yaml:"specTopic"`
```
[the remark this answers is not in the document]
Comment (skeeey) [reply pkg/cloudevents/generic/types/types.go]: feels `topics.spec` is better, updated

**Review by morvencao — COMMENTED** [review]
🚀 
/lgtm

**Pull request merged**